libraries/helper.rb

```diff
@@ -46,14 +46,24 @@ def omnibus_env
       node.run_state[:omnibus_env] ||= Hash.new { |hash, key| hash[key] = [] } # ~FC001
     end
 
+    # TODO: this method should just go away
     def omnibus_toolchain_enabled?
-      # We don't have an omnibus toolchain for windows yet.
-      !windows?
+      true
+    end
+
+    def toolchain_install_dir
+      if windows?
+        windows_safe_path_join(ENV['SYSTEMDRIVE'], 'opscode', node['omnibus']['toolchain_name'])
+      else
+        "/opt/#{node['omnibus']['toolchain_name']}"
+      end
     end
 
     def build_user_shell
-      if omnibus_toolchain_enabled?
-        "/opt/#{node['omnibus']['toolchain_name']}/bin/bash"
+      if windows?
+        windows_safe_path_join(toolchain_install_dir, 'embedded', 'bin', 'usr', 'bin', 'bash')
+      elsif omnibus_toolchain_enabled?
+        ::File.join(toolchain_install_dir, 'bin', 'bash')
       else
         '/usr/local/bin/bash'
```

Comment:
This can also go away with the `omnibus_toolchain_enabled?` method - just a note.